src/bakery_manager_mobile/lib/pages/task_details_page.dart

```diff
@@ -189,7 +224,7 @@ class _TaskDetailPageState extends State<TaskDetailPage> {
                         text: 'Assigned on: ',
```

Comment:
consideration: if you change a task and change the employee, should the Assignment time update?